lib/ansible/modules/cloud/ovirt/ovirt_disk.py

```diff
@@ -192,6 +192,7 @@
             - I(True) if the disk should be activated.
         version_added: "2.8"
         type: bool
+        default: True
```

Comment:
Can you please change the behavior so the True is used only in case the disk is being created. In case it should be updated don't use True, but rather don't do anywhing if user didn't specify activate parameter.